### PR TITLE
Update and rename MacOSTrainingGuide.md to MacOS-LinuxTrainingGuide.md

### PR DESCRIPTION
**Pull request by ajstinn2**
-  Language and name changed to include Linux.
- Changing install instructions to use the new PKG installer.
- Commands and language updated to reflect 3.8.0
- Better linking to the downloads page.

### CODE REVIEW

File: MacOS-LinuxTrainingGuide.md [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
+# StackHawk Developer Training: Getting Started With HawkScan - MacOS / Linux Training Guide
 
 This guide will be used in tandem with the live training given by the StackHawk Team. It can also be used in a self-paced fashion to walk through the steps of getting started with HawkScan. Throughout the training, we will reference specific commands that must be entered in your terminal. Those commands are outlined below, with detailed descriptions.
 
 ## Step 1: _Installing HawkScan_
-There are multiple different methods for [downloading](https://docs.stackhawk.com/stackhawk-cli/#installation-overview) the HawkScan CLI tool. But for the purpose of this training, we will install HawkScan via HomeBrew. Run the following brew command in your terminal to download and install the latest version of HawkScan.
+There are multiple different methods for [downloading](https://docs.stackhawk.com/download.html) the HawkScan CLI tool. But for the purpose of this training, we will install HawkScan via our PKG installer. Run the following curl command in your terminal to download and install the latest version of HawkScan.
```
Comment (kcberg) [inline MacOS-LinuxTrainingGuide.md]: There are multiple install methods and [downloads](https://docs.stackhawk.com/download.html) of the HawkScan CLI tool.

**Review by kcberg — CHANGES_REQUESTED** [review]
words

File: MacOS-LinuxTrainingGuide.md (hunk 1 above)
Comment (ajstinn2) [reply MacOS-LinuxTrainingGuide.md]: @kcberg Language fixed to better reflect install methods and download options

**Review by azconger — DISMISSED** [review]
LGTM!

**Review by kcberg — APPROVED** [review]

**Pull request merged**